3-probability/06-normal-approx/notes.qmd

```diff
@@ -0,0 +1,841 @@
+---
+title: "Continuous Distributions and Normal Approximations"
+subtitle: "Connections to boxes, continuous distributions, and a fundamental result"
+image: images/front-image.png
+format:
+  html: default
+  pdf: default
+editor_options: 
+  chunk_output_type: console
+---
+
+:::{.lo .content-hidden unless-profile="staff-site"}
```

Comment:
Awesome job getting these articulated!

Reply:
Thanks :) I had to because I wasn't sure what had go in here.